adapter/package.json

```diff
@@ -44,7 +43,7 @@
         "classnames": "^2",
         "moment": "^2",
         "prop-types": "^15",
-        "react": "^16.8",
+        "react": "^16.8 || ^18",
```

Comment:
Should this be just `^18`?  If not, maybe all dependencies (including `shell`, which isn't a peerdep) should use the union `^16.8 || ^18`?  That would allow an app to specify `^16.8` and yarn would resolve to a single React version `16.8` since that would satisfy both.  Otherwise if we're forcing a minimum version of 18 we could say `^18` and drop the `^16.8` in this peer dep?

Maybe easier to explain on a quick call or chat next week.  It may be fine to leave this as-is, just curious what the reasoning is.

Reply:
yeah it should have been just ^18, updated it now - my reasoning was related to my question on why react is not a peer dependency of the shell. I thought the shell and adapter could live with either react@16 or 18, but that's not possible anyhow as the changes in the shell would only work in React@18.